Orthtree/doc/Orthtree/Concepts/OrthtreeTraits.h

```diff
@@ -18,45 +17,109 @@ class OrthtreeTraits
   /// \name Types
   /// @{
 
-  typedef unspecified_type Dimension; ///< Dimension type (see `CGAL::Dimension_tag`).
-  typedef unspecified_type Bbox_d; ///< Bounding box type.
-  typedef unspecified_type FT; ///< The number type of the %Cartesian coordinates of types `Point_d`
-  typedef unspecified_type Point_d; ///< Point type.
-  typedef unspecified_type Sphere_d; ///< The sphere type for neighbor queries.
+  using Dimension = unspecified_type; ///< Dimension type (see `CGAL::Dimension_tag`).
+  using FT = unspecified_type; ///< The number type of the %Cartesian coordinates of types `Point_d`
+  using Point_d = unspecified_type; ///< Point type.
+  using Bbox_d = unspecified_type; ///< Bounding box type. Must be constructible from a pair of `Point_d` types.
 
   /*!
     A random access iterator type to enumerate the
     %Cartesian coordinates of a point.
   */
-  typedef unspecified_type Cartesian_const_iterator_d;
-  typedef std::array<FT, Dimension::value> Array; ///< Array used for easy point constructions.
+  using Cartesian_const_iterator_d = unspecified_type;
 
-  typedef unspecified_type Adjacency; ///< Specify the adjacency directions
+  /*!
+   * \brief The data type contained by each node.
+   */
+  using Node_data = unspecified_type;
 
   /*!
-    Functor with an operator to construct a `Point_d` from an `Array` object.
-  */
-  typedef unspecified_type Construct_point_d_from_array;
+   * \brief Number-type which can take on values indicating adjacency directions.
+   *
+   * Must be able to take on values ranging from 0 to the number of faces of the (hyper)rectangle type, equivalent to 2 * D.
+   */
+  using Adjacency = unspecified_type; ///< Specify the adjacency directions
 
   /*!
-    Functor with an operator to construct a `Bbox_d` from two `Array` objects (coordinates of minimum and maximum points).
-  */
-  typedef unspecified_type Construct_bbox_d;
+   * \brief Functor with an operator to create the bounding box of the root node.
+   *
```

Comment:
Here the signatures of all the functors must be added.

Reply:
done